packages/styles/components/_c.hero.scss

```diff
@@ -0,0 +1,261 @@
+        padding-left: $mu800;
+        padding-right: $mu200;
+      }
+
+      @include set-from-screen('xl') {
+        padding-left: 15rem;
+        padding-right: 15rem;
+      }
+
+      @include set-from-screen('xl-medium') {
+        padding-left: 13rem;
+        padding-right: 13rem;
+      }
+
+      @include set-from-screen('xl-large') {
+        padding-left: 22rem;
+        padding-right: 22rem;
+      }
+
+
+
+      .mc-hero__header {
+        @include set-from-screen('m-medium') {
+          @include set-flexy-width(1, 1);
+        }
+
+        @include set-from-screen('m-medium') {
+          padding-right: 5rem;
+          padding-left: 5rem;
+        }
+
+        @include set-from-screen('xl') {
+          padding-right: 6.5rem;
+          padding-left: 6.5rem;
+        }
+
+        @include set-from-screen('xl-large') {
+          padding-right: 9.75rem;
+          padding-left: 9.75rem;
+        }
+      }
+
+      .mc-hero__content {
+        justify-content: center;
+        text-align: center;
+
+        @include set-from-screen('m-medium') {
+          padding-right: $mu500;
+          padding-left: $mu500;
+        }
+
+        @include set-from-screen('l-medium') {
+          padding-right: 6.5rem;
+          padding-left: 6.5rem;
+        }
+
+        @include set-from-screen('xl-medium') {
+          padding-right: 13.5rem;
+          padding-left: 13.5rem;
+        }
+
+        @include set-from-screen('xl-large') {
+          padding-right: 22rem;
+          padding-left: 22rem;
+        }
+      }
+    }
+
+    &--with-box {
+      @include set-from-screen('s') {
+        padding: $mu250 $mu125 $mu900 $mu125;
+      }
+
+      @include set-from-screen('s-medium') {
+        padding: $mu250 $mu125 $mu900 $mu125;
+      }
+
+      @include set-from-screen('s-large') {
+        padding: $mu250 $mu125 $mu900 $mu125;
+      }
+
+      @include set-from-screen('s-xlarge') {
+        padding: $mu250 $mu125 $mu900 $mu125;
+      }
+
+      @include set-from-screen('m') {
+        padding: $mu400 $mu200 $mu900 $mu200;
+      }
+
+      @include set-from-screen('m-medium') {
+        padding: $mu300 $mu300 $mu900 $mu300;
+      }
+
+      @include set-from-screen('l') {
+        padding: $mu300 $mu300 $mu900 $mu300;
+      }
+
+      @include set-from-screen('xl') {
+        padding: $mu400 $mu200 10rem $mu200;
+      }
+
+      @include set-from-screen('xl-medium') {
+        padding: $mu400 $mu700 10rem $mu700;
+      }
+
+      @include set-from-screen('xl-large') {
+        padding: $mu400 $mu700 10rem $mu700;
+      }
+
+      @include set-from-screen('xxl') {
+        padding: $mu400 $mu250 11rem $mu250;
+      }
+
+      .mc-hero__header {
+        @include set-from-screen('m-medium') {
+          @include set-flexy-width(9, 12);
+        }
+      }
+
+      .mc-hero__content {
+        @include set-from-screen('m-medium') {
+          @include set-flexy-width(10, 12);
+        }
+      }
+    }
+  }
```

Comment:
Since this is not going to be committed now, can you please remove this part from the commit?